itest/utils.go

```diff
@@ -147,7 +147,7 @@ func MineBlocks(t *testing.T, client *rpcclient.Client,
 	var blockHashes []*chainhash.Hash
 
 	switch backend.(type) {
-	case rpcclient.BitcoindVersion:
+	case *rpcclient.BitcoindVersion:
```

Comment:
Maybe we should add a comment here? Otherwise someone looking at this will definitely think one of them is wrong...

Reply:
aren't both meant to be pointers?

Reply:
Nope, apparently they're not, which is very weird...